api/specs.proto

```diff
@@ -293,6 +293,23 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	enum Isolation {
+		option (gogoproto.goproto_enum_prefix) = false;
+
+		// DEFAULT uses whatever default value from the container runtime
+		DEFAULT = 0 [(gogoproto.enumvalue_customname) = "ContainerIsolationDefault"];
```

Comment:
These need to be `ISOLATION_DEFAULT`, `ISOLATION_PROCESS` and `ISOLATION_HYPERV`.

Reply:
Sorry about that, just amended my last commit